### PR TITLE
chore: Update PR template to add note about json mocks

### PR DESCRIPTION
**Pull request by nikhilvenkatraman-okta**
## Description:

- Update PR template to add note about json mocks

## PR Checklist

- [x] Have you verified the basic functionality for this change?

### Reviewers:

@jmelberg-okta @aarongranick-okta 

### Issue:

- [OKTA-402435](https://oktainc.atlassian.net/browse/OKTA-402435)

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE/pr_external.md [hunk 1]
```diff
@@ -10,6 +10,8 @@
 
 - [ ] Added e2e tests
 
+- [ ] Added json mocks for any new views?
```
Comment (jmelberg-okta) [inline .github/PULL_REQUEST_TEMPLATE/pr_external.md:13]: Within our `test` directory, we have `e2e`, `testcafe`, and `unit` types. Here, let's couple this to `testcafe` to be extremely verbose:

 ```suggestion
- [ ] Added testcafe tests with JSON mocks for any new views
```

**Review by jmelberg-okta — APPROVED** [review]
One suggestion!

File: .github/PULL_REQUEST_TEMPLATE/pr_external.md (hunk 1 above)
Comment (jmelberg-okta) [reply .github/PULL_REQUEST_TEMPLATE/pr_external.md:13]: Also - can we cleanup the double newline spaces between these items and remove the `?` from the **unit** label?

Comment (nikhilvenkatraman-okta) [reply .github/PULL_REQUEST_TEMPLATE/pr_external.md]: Fixed

**Review by pradeepdewda-okta — APPROVED** [review]

**Pull request closed without merging**